### DISCUSSION TITLE
Fix ExtendedASCII

### DISCUSSION THREAD

**Discussion by KernelFox0** [CosmosOS/Cosmos#2699] [Ideas]
ExtendedASCII is broken in current build of cosmos but it worked before.
Like making this possible again to print the █ character (ASCII 219):
```cs
Console.OutputEncoding = CosmosEncodingProvider.Instance.GetEncoding(437);
```

**Comment by 9xbt** [comment]
I think its a .NET issue. I think they updated the compiler to do something and theres a method that's being called that isn't plugged

**Comment by KernelFox0** [comment]
Maybe. The error is:
```
Need plug for: System.Boolean  Interop+Kernel32.SetConsoleOutputCP(System.Int32)(Plug Signature: System_Boolean__Interop_Kernel32_SetConsoleOutputCP_System_Int32_ ).
```
So that function is not plugged, right?

**Comment by ADev531** [comment]
im doing
<code>
 Encoding.RegisterProvider(CosmosEncodingProvider.Instance);
Console.OutputEncoding = CosmosEncodingProvider.Instance.GetEncoding(437);
</code>

why not working
